File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        r=[]
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                x=nums[i]
                y=nums[j]
                if x+y==target:
                    r.append(i)
                    r.append(j)
                    return r
        return r
                    
        

File: 1-two-sum/1-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> res;
        vector<int>r;
        r.push_back(-1);
        r.push_back(-1);
        for(int i=0;i<nums.size();i++)
        {
            for(int j=i+1;j<nums.size();j++)
            {
                if(nums[i]+nums[j]==target)
                {
                    res.push_back(i);
                    res.push_back(j);
                    return res;
                }
            }
        }
        
        return r;
    }
};